django/db/backends/oracle/oracledb_any.py

```diff
@@ -0,0 +1,21 @@
+import warnings
+
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.deprecation import RemovedInDjango60Warning
+
+try:
+    import oracledb
+
+    is_oracledb = True
+except ImportError as e:
+    try:
+        import cx_Oracle as oracledb  # NOQA
+
+        warnings.warn(
+            "cx_Oracle is deprecated. Please use oracledb instead.",
+            RemovedInDjango60Warning,
+            stacklevel=2,
+        )
+        is_oracledb = False
+    except ImportError:
+        raise ImproperlyConfigured("Error loading oracledb module: %s" % e)
```

Comment:
```suggestion
        raise ImproperlyConfigured("Error loading cx_Oracle module: %s" % e)
```

Reply:
The error `e` is raised when importing oracledb so the traceback is for "Error loading oracledb module".

"Error loading oracledb module" is better in most cases.  "Error loading cx_Oracle module" may be better when oracledb is not installed and cx_Oracle is installed but unloadable. But displaying "Error loading oracledb module" is also acceptable here since people will just install oracledb when seesing the message.

|  |oracledb is not installed|oracledb is installed but unloadable|
|---|----|----|
|__cx_Oracle is not installed__|"Error loading oracledb module" is better.|"Error loading oracledb module" is better.|
|__cx_Oracle is installed but unloadable__|"Error loading cx_Oracle module" may be better. But "Error loading oracledb module" is also acceptable to me.|"Error loading oracledb module" is better.|